src/routes/helpers/transformers.js

```diff
@@ -1,6 +1,7 @@
 import SchemaHelper from '../../helpers/SchemaHelper'
 import { warning } from '../../utils/logger'
 import { isDateTime } from 'neo4j-driver/lib/temporal-types'
+import { schema } from '../../schema'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['schema' is defined but never used.](https://app.codacy.com/manual/trompamusic/ce-api/pullRequest?prid=5229678)